transmart-core-db-tests/src/groovy/org/transmartproject/db/dataquery/highdim/snp_lz/SnpLzTestData.groovy

```diff
@@ -208,6 +208,16 @@ class SnpLzTestData {
                         it.chromosome == annotation.chromosome }
             assert snpInfo != null
 
+            // Choose value of minorAllele based on the allele count, to vary its
+            // value: to generate test cases for both 'A1' and 'A2'.
+            long a1Count = 0
+            long a2Count = 0
+            gtss.each {
+                a1Count += it.findAll( a1 ).size()
+                a2Count += it.findAll( a2 ).size()
```

Comment:
I'm not sure what's happening here. Up we see that gtss has the format `['T T', 'T A', ...]`. Then `it` inside the closure must be something like `'T T'`. OTOH, 'a1' is a String. So you get something like `'T T'.findAll { 'A' }.size()`, which always returns the number of characters in the first string, that is 3.

This function is also inconsistent: you left `def a1 = annotation.alt` at the top, but then `a1` may not be the alternative. It seems to me that the choice about which allele will be minor one should be made before `a1` and `a2` are assigned.